var/spack/repos/builtin/packages/py-wget/package.py

```diff
@@ -12,7 +12,7 @@ class PyWget(PythonPackage):
     Download the file for your platform. If you're not sure which to choose,
     learn more about installing packages."""
 
-    homepage = "http://bitbucket.org/techtonik/python-wget/"
+    homepage = "https://pypi.org/project/wget/"
```

Comment:
You can remove this line, it's the default set by `pypi`

Reply:
This should be fixed.